src/Tools/dotnet-trace/CommandLine/Commands/CollectCommand.cs

```diff
@@ -137,14 +138,24 @@ private static async Task<int> Collect(CancellationToken ct, IConsole console, i
                         return (int)ReturnCode.ArgumentError;
                     }
 
-                    collectRundownEvents = selectedProfile.Rundown;
+                    rundownKeyword = selectedProfile.RundownKeyword;
+                    retryStrategy = selectedProfile.RetryStrategy;
 
                     Profile.MergeProfileAndProviders(selectedProfile, providerCollection, enabledBy);
                 }
 
                 if (rundown.HasValue)
                 {
-                    collectRundownEvents = rundown.Value;
+                    if (rundown.Value)
+                    {
+                        rundownKeyword = EventPipeSession.DefaultRundownKeyword;
```

Comment:
I assume we'd want to merge this with any rundown keyword that came from the profile. Its possible the profile specified some additional keyword bits.

Reply:
Agreed